src/sinks/humio/metrics.rs

```diff
@@ -102,7 +102,7 @@ impl SinkConfig for HumioMetricsConfig {
             tls: self.tls.clone(),
             timestamp_nanos_key: None,
             acknowledgements: Default::default(),
-            timestamp_key: Default::default(),
+            timestamp_key: "timestamp".to_string(),
```

Comment:
Ah, I spoke with Jesse about this one. I believe he actually suggested hard coding it to "timestamp" because that is what humio expects ? If that is still the way to go, I can add a comment here so that is more clear.

Reply:
Yeah, that sounds good